.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         uses: bahmutov/npm-install@c5482d70ec8706408996e31ac94075030694993f # v1.8.32
```

Comment:
🤔 It seems this dep might need an update as well, according to this check log:
https://github.com/hashicorp/dev-portal/actions/runs/7889328354

Reply:
Good catch! Corrected here 533820d67116101334aa472908c92a1ff680531b